en/mkdocs.yml

```diff
@@ -1414,7 +1414,9 @@ nav:
             - Configure PingFederate as A Key Manager: administer/key-managers/configure-pingfederate-connector.md
             - Configure ForgeRock as a Key Manager: administer/key-managers/configure-forgerock-connector.md
             - Configure a Custom Key Manager: administer/key-managers/configure-custom-connector.md
+            - Configure the Global Key Manager: administer/key-managers/configure-global-key-manager.md
         - Advanced Configurations: administer/advanced-configurations.md
+        - Manage Role based access control for the Admin portal: administer/role-based-access-control.md
```

Comment:
Shall we check how this got added? This doesn't seem to be a part of the Global Key Manager docs

Reply:
It has been added from the cherry pick. Removed that line